algopy_testing/src/algopy/primitives/uint64.py

```diff
@@ -46,7 +50,7 @@ def __lt__(self, other: int | UInt64) -> bool:
     def __bool__(self) -> bool:
         return self.value != 0
 
-    def __add__(self, other: int | UInt64) -> UInt64:
+    def __add__(self, other: int | UInt64 | BigUInt) -> UInt64:
```

Comment:
To resolve the type checking errors, we should instead update `BigUInt.__radd__` method (and other applicable dunder methods) to include `UInt64` in it's type hints. This will ensure our implementation and the stubs match